python/ray/_private/ray_logging/formatters.py

```diff
@@ -27,7 +28,7 @@ def generate_record_format_attrs(
 
     for key, value in record.__dict__.items():
         # Both Ray and user-provided context are stored in `record_format`.
-        if key not in LOGRECORD_STANDARD_ATTRS:
+        if key not in LOGRECORD_STANDARD_ATTRS or key == LogKey.CREATED:
```

Comment:
It's better to include created in `LOGGER_FORMAT`. I am not sure whether this will break anything.